bulk_grades/views.py

```diff
@@ -93,8 +93,13 @@ def post(self, request, course_id, *args, **kwargs):
             data = self.processor.status()
             data['error_messages'] = []
             for error_message in self.processor.error_messages:
-                line_numbers = ', #'.join(str(line_number+1) for line_number in self.processor.error_messages[error_message])
-                data['error_messages'].append(f'{error_message} on lines (#{line_numbers})')
+                line_numbers = ', '.join(str(line_number+1) for line_number in self.processor.error_messages[error_message])
+                new_message = '{error_message} on line{is_plural} {line_numbers};'.format(
```

Comment:
I would move the semicolon to `frontend-app-gradebook`, in case we have multiple other errors that we need to join, let Gradebook deal with how to join them

Reply:
The solution all of us agree were ended with **;** for each message.